README.md

```diff
@@ -67,9 +67,9 @@ For help, please consider the following venues (in order):
 For all Kubernetes releases, we recommend installing the latest VPC CNI release. The following table denotes our minimum recommended
 VPC CNI version for each actively supported Kubernetes release.
 
-| Kubernetes Release |    1.26    |   1.25    |    1.24    |    1.23    |    1.22    |
-| ------------------ | ---------- | ----------| ---------- | ---------- | ---------- |
-|  VPC CNI Version   |  v1.12.0+  | v1.11.4+  |  v1.9.3+   |  v1.8.0+   |  v1.8.0+   |
+| Kubernetes Release | 1.27     | 1.26     | 1.25     | 1.24    | 1.23    | 1.22    |
```

Comment:
From `aws eks describe-addon-versions --addon-name vpc-cni --kubernetes-version 1.27`, it looks like the oldest supported version we claim to support is actually `v1.12.5`, so I think we can choose that here for 1.27. Also, since 1.22 is out of support, I think we can remove that column.